Assets/Scripts/MarchingCubes/MarchingCubes.cs

```diff
@@ -10,42 +11,37 @@ public class MarchingCubes
 {
     readonly ComputeShader meshGenerator;
     readonly float threshold;
-    readonly int resolution;
-    readonly float radius;
+    readonly float diameter;
     readonly int frequency;
     readonly float amplitude;
-    readonly float bottomLevel;
-
-    Mesh mesh;
+    public int chunkResolution;
 
     /// <summary>
     /// Initializes the MarchingCubes script
     /// </summary>
     /// <param name="mesh"></param>
```

Comment:
Seems to be some old documentation. Either document the parameters properly or just remove them and just keep the method summary.

Reply:
fixed